api/rosetta/data.go

```diff
@@ -14,11 +14,14 @@
 
 package rosetta
 
+// Data implements the Rosetta Data API specification.
+// See https://www.rosetta-api.org/docs/data_api_introduction.html
 type Data struct {
 	config   Configuration
 	retrieve Retriever
 }
 
+// NewData creates a new instance of the Data API.
```

Comment:
`... using the given configuration to answer configuration queries and the given retriever to answer blockchain data queries.`  or something like that?

I like to always mention the given parameters in the sentence with `the given x, and the given y`.